src/libraries/Microsoft.PowerFx.Core/Public/IntellisenseOperations.cs

```diff
@@ -91,6 +91,67 @@ public bool ValidateInvocation(string functionName, IReadOnlyList<TexlNode> args
             return false;
         }
 
+        /// <summary>
+        /// Checks whether function's n-th argument is row scoped.
+        /// </summary>
+        /// <param name="functionIdentifier"></param>
+        /// <param name="argNum"></param>
+        /// <returns></returns>
+        public bool IsRowScopeArg(Identifier functionIdentifier, int argNum)

```

Comment:
The assumption here is that if this is true, then use arg0's type from syntax tree? (per coercions, etc, that may not be the final type from binder).

Reply:
As we discussed it can only be arg. num 0, so I guess this makes sense. New semantics of the function reflects that we're checking whether argument can be row-scope under some overload, not necessarily for all of them (e.g., as in case of `Sum`).